cc.config.json

```diff
@@ -546,6 +546,14 @@
             "internal": false,
             "dynamic": true
         },
+        "EDITOR_NOT_IN_PREVIEW" : {
+            "comment": "Run in editor but not in editor preview.",
+            "type": "boolean",
+            "value": false,
+            "ccGlobal": true,
```

Comment:
ccGlobal is for 2.x compliant implementations and should be set to false here.

Reply:
done